.golangci.yml

```diff
@@ -0,0 +1,60 @@
+# This file configures github.com/golangci/golangci-lint.
+
+run:
+  timeout: 3m
+  tests: true
+  # default is true. Enables skipping of directories:
+  #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
+  skip-dirs-use-default: true
+
+service:
+  golangci-lint-version: 1.49.0
+
+linters:
+  disable-all: true
+  enable:
+    - whitespace # Tool for detection of leading and trailing whitespace
+    - wsl # Whitespace Linter - Forces you to use empty lines!
+    - wastedassign # Finds wasted assignment statements.
+    - unconvert # Unnecessary type conversions
+    - tparallel # detects inappropriate usage of t.Parallel() method in your Go test codes
+    - thelper # Detects golang test helpers without t.Helper() call and checks the consistency of test helpers
+    - stylecheck # Stylecheck is a replacement for golint
+    - prealloc # Finds slice declarations that could potentially be pre-allocated
+    - predeclared # Find code that shadows one of Go's predeclared identifiers
+    - nolintlint # Ill-formed or insufficient nolint directives
+    - nlreturn # Checks for a new line before return and branch statements to increase code clarity
+    - misspell # Misspelled English words in comments
+    - makezero # Finds slice declarations with non-zero initial length
+    - lll # Long lines
+    - importas # Enforces consistent import aliases
+    - gosec # Security problems
+    - gofmt # Whether the code was gofmt-ed
+    - goimports # Unused imports
+    - goconst # Repeated strings that could be replaced by a constant
+    - forcetypeassert # Finds forced type assertions
+    - dogsled # Checks assignments with too many blank identifiers (e.g. x, , , _, := f())
+    - dupl # Code clone detection
+    - errname # Checks that sentinel errors are prefixed with the Err and error types are suffixed with the Error.
```

Comment:
Same as the previous comment, comment should not end in a period

Reply:
Fixed